server/package.json

```diff
@@ -13,7 +15,7 @@
     "url": "git://github.com/pol-is/polisServer.git"
   },
   "author": "",
-  "license": "",
+  "license": "AGPL-3.0",
```

Comment:
specify the license used by the parent project to fix the `NO LICENSE FOUND` warning in the build logs